devices/Ssd13xx/Ssd1306.cs

```diff
@@ -17,6 +18,13 @@ public class Ssd1306 : Ssd13xx
         /// Default I2C bus address.
         /// </summary>
         public const byte DefaultI2cAddress = 0x3C;
+		
+		/// <summary>
```

Comment:
You have some space/tabs/alignment issue here. ctrl + k + d in Visual Studio should fix all those.
Also, we're now using a linter. See other bindings which already include it. It's StyleCop (old way). You can add the nuget, then add the section in the nfproj (sample not needed), copy/paste the StyleCop file. And it will as well catch up style issues like this one.